akka-distributed-data/src/main/scala/akka/cluster/ddata/Flag.scala

```diff
@@ -8,11 +8,16 @@ object Flag {
    * `Flag` that is initialized to `false`.
    */
   val empty = new Flag(false)
-  def apply(): Flag = empty
+
+  val Disabled = empty
```

Comment:
we try to be explicit about the return type in public API:s, so add `: Flag`

Reply:
Ok